guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-hosts.adoc

```diff
@@ -1,14 +1,58 @@
 [id="deploying-a-custom-ssl-certificate-to-hosts_{context}"]
-
 = Deploying a Custom SSL Certificate to Hosts
 
-After you configure {ProductName} to use a custom SSL certificate, you must also install the `katello-ca-consumer` package on every host that is registered to this {ProductName}.
+After you configure {Project} to use a custom SSL certificate, you have to provide this certificate used by `subscription-manager` on hosts registered to {Project}.
 
 .Procedure
-
-* On each host, install the `katello-ca-consumer` package:
+* On each host, update the certificates:
```

Comment:
I assume that only one certificate is being updated on each host, so you might want to use the singular: 
"Update the SSL certificate on each host:"

Reply:
Applied.